about.html

```diff
@@ -110,7 +102,27 @@ <h4>See the past CC Global Summits<hr></h4>
             </div>
         </div>
     </section>
-
+    <section class="partner">
+        <p>Partner<hr></p>
+        <div class="images-partner">
+            <img src="img/mozilla 1.png" alt="mozile logo" width="109" height="31">
+            <img src="img/googlekorea 1.png" alt="Google logo" width="109" height="31">
+            <img src="img/naver 1.png" alt="Naver logo" width="109" height="31">
+            <img src="img/daumkakaologo 1.png" alt="daumkakao logo" width="109" height="31">
+            <img src="img/airbnb 1.png" alt="Airbnb logo" width="109" height="31">
+        </div>
```

Comment:
- [ ] [OPTIONAL] I am not sure why you have this here but if this is how it was implemented in the `index.html`, I think you should consider using a `ul`, `li` to render this list of similar items. It makes the styling and grouping easy :smile:

Reply:
Great, I'll work on it once I'm done with everything!